partner_bank_sort_code/models/res_bank.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResBank(models.Model):
+
+    _inherit = 'res.bank'
+
+    sort_code = fields.Char()
+
+    _sql_constraints = [(
+        "sort_code_unique",
+        "EXCLUDE (sort_code WITH =) WHERE (active = True)",
```

Comment:
@rousseldenis  First time I read this syntax but it looks like you exclude the active record from the unique contrains. I've to check the postgresql documentation.

Reply:
Yes, the term is ambiguous. I've added tests and comments in order to be consistent

Reply:
@rousseldenis Great! Also tested on my side....